index.html

```diff
@@ -1,576 +1,589 @@
+          padding: .5em;
+          border: .5em;
+          border-left-style: solid;
+          border-color: #52e052;
+          page-break-inside: avoid;
+          margin: 1em auto;
+      }
     </style>
-</head>
-
-<body>
+  </head>
+  <body>
     <header id="header">
-        <aside>
-            <ul id="navbar">
-                <li><a href="#scope">Scope</a></li>
-                <li><a href="#deliverables">Deliverables</a></li>
-                <li><a href="#success-criteria">Success Criteria</a></li>
-                <li><a href="#coordination">Coordination</a></li>
-                <li><a href="#participation">Participation</a></li>
-                <li><a href="#communication">Communication</a></li>
-                <li><a href="#decisions">Decision Policy</a></li>
-                <li><a href="#patentpolicy">Patent Policy</a></li>
-                <li><a href="#licensing">Licensing</a></li>
-                <li><a href="#about">About this Charter</a></li>
-            </ul>
-        </aside>
-        <p>
-            <a href="https://www.w3.org/">
-                <img src="https://www.w3.org/Icons/w3c_home" alt="W3C" height="48" width="72">
-            </a>
-        </p>
+      <aside>
+        <ul id="navbar">
+          <li><a href="#background">Background</a></li>
+          <li><a href="#scope">Scope</a></li>
+          <li><a href="#deliverables">Deliverables</a></li>
+		  <li><a href="#success-criteria">Success Criteria</a></li>
+          <li><a href="#coordination">Coordination</a></li>
+          <li><a href="#participation">Participation</a></li>
+          <li><a href="#communication">Communication</a></li>
+          <li><a href="#decisions">Decision Policy</a></li>
+          <!-- <li><a href="#patentpolicy">Patent Policy</a></li> -->
+          <li><a href="#patentpolicy">Patent Disclosures</a></li>
+          <li><a href="#licensing">Licensing</a></li>
+          <li><a href="#about">About this Charter</a></li>
+        </ul>
+      </aside>
+      <p>
+        <a href="https://www.w3.org/"><img alt="W3C" height="48" src="https://www.w3.org/Icons/w3c_home" width="72"></a>
+      </p>
     </header>
 
 
     <main>
-        <h1 id="title">
-            RDF Dataset Canonicalization and Hash Working Group Charter
-        </h1>
+      <h1 id="title">
+        <i class=todo>PROPOSED</i> RDF Dataset Canonicalization and Hash Working Group Charter
+      </h1>
+
+      <p class="mission">
+        The <strong>mission</strong> of the <a href="https://www.w3.org/groups/wg/rch">RDF Dataset Canonicalization and Hash Working Group</a> is to maintain the specifications previously published by this group. That work included a standard to uniquely and deterministically calculate a hash of RDF Datasets for use cases such as <a href="explainer.html#detectingChanges">Detecting changes in Datasets</a>, and was based  on an <a href="explainer.html#canonicalization">RDF Dataset Canonicalization</a> algorithm.
```

Comment:
```suggestion
        The <strong>mission</strong> of the <a href="https://www.w3.org/groups/wg/rch">RDF Dataset Canonicalization and Hash Working Group</a> is to maintain the specifications previously published by this group. That work included a standard to uniquely and deterministically calculate a hash of RDF Datasets for use cases such as <a href="https://www.w3.org/TR/rch-explainer/#detectingChanges">Detecting changes in Datasets</a>, and was based  on an <a href="https://www.w3.org/TR/rch-explainer/#canonicalization">RDF Dataset Canonicalization</a> algorithm.
```
Similar to @philarcher 's suggestion.